src/ReactiveUI.XamForms/RoutedViewHost.cs

```diff
@@ -94,7 +94,16 @@ public RoutedViewHost()
                         }
                         else
                         {
-                            await this.PushAsync(x);
+                            bool animated = true;
+                            var type = x.GetType();
+                            Attribute[] attrs = Attribute.GetCustomAttributes(type);
```

Comment:
Can we get a unit test for this one at all? Know it's a been a while since you did the PR.

Also I think you can use the type to get the Attribute..

eg 

var attribute = type.GetCustomAttribute<DisableAnimationAttribute>();
if (attribute != null) type thing.